server/config/parse-options.js

```diff
@@ -4,11 +4,22 @@ var path = require('path')
 
 var defaultsDeep = require('lodash').defaultsDeep
 var log = require('npmlog')
+var extend = require('extend')
```

Comment:
I think you can use lodash’s [defaultsDeep](https://lodash.com/docs#defaultsDeep) instead of `extend`?